src/pages/jobSearch/JobSearchPage.tsx

```
@@ -14,35 +14,36 @@ import { useScrapCheckQuery } from '@hook/scrap/useScrapCheckQuery';
 import { useScrapRecruitMutation } from '@hook/scrap/recruit/useScrapRecruitMutation';
 import { useQueryClient } from '@tanstack/react-query';
 import { ReactTagManager } from 'react-gtm-ts';
+import { useLocation } from 'react-router-dom';
 
 const sortOptions = ['마감 임박순', '마감 여유순'];
 
+const location = useLocation();
```

ESLint

[error] React Hook "useLocation" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)